App/Containers/PhotoViewerScreen.js

```diff
@@ -186,7 +187,7 @@ const mapStateToProps = (state, ownProps) => {
 
 const mapDispatchToProps = (dispatch) => {
   return {
-    share: (item) => { dispatch(IpfsActions.shareImageRequest('all', item.hash)) },
+    share: (item) => { dispatch(IpfsActions.shareImageRequest(Config.ALL_THREAD_NAME, item.hash)) },
```

Comment:
Didn't we decide this `IpfsActions.shareImageRequest` doesn't exist anymore?

Reply:
oh yeah. what can be deleted here? `share`, `authorShare`, `cancelAuthoringShare`?

Reply:
Looks like `share` and `cancelAuthoringShare` can be removed here.